modules/showTime.js

```diff
@@ -0,0 +1,7 @@
+import { DateTime } from './luxon.js';
+
+export default function showTime() {
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that you make use of arrow functions because it a project requirement.

## EXAMPLE
export default () => console.log("Hello World.");

// If you want named exports, you have to export it seperately
const showTime = () => console.log("This is an app.");
export default showTime;